demo/molecule/modal/playground

```diff
@@ -0,0 +1,320 @@
+const IconClose = () => <span>X</span>
+class ScrollModal extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      open: false
+    }
+  }
+
+  openModal = () =>  {
+    this.setState({
+      open: true
+    })
+  }
+
+  closeModal = () => {
+    this.setState({
+      open: false
+    })
+  }
+
+  render() {
+    return (
+      <div>
+        <button type="button" onClick={this.openModal}>
+          Open modal
+        </button>
+        <MoleculeModal
+          isOpen={this.state.open}
+          closeOnOutsideClick
+          closeOnEscKeyDown
+          iconClose={<IconClose />}
+          fitWindow
+          header={<strong>My new brand modal</strong>}
+          onClose={this.closeModal}
+        >
+          <div>
+              <p>
```

Comment:
you could create a component with this paragraph

Reply:
done!